library-app/src/components/Layout/MainLayout.tsx

```diff
@@ -0,0 +1,24 @@
+import Footer from '../Footer/Footer'
+import Header from '../Header/Header'
+import Sidebar from '../Sidebar/Sidebar'
+import styles from './MainLayout.module.css'
+
+export interface IMainLayout {
+  children: React.ReactNode
+}
+
+// eslint-disable-next-line react/prop-types
+const MainLayout: React.FC<IMainLayout> = ({ children }) => {
```

Comment:
There is a class PropsWithChildren in react with which you can avoid this eslint-disable